src/app/application-form/application-form.component.ts

```diff
@@ -173,64 +180,60 @@ export class ApplicationFormComponent implements OnInit {
 
   get name() {
     return this.applicationForm
-      .get('customerInformation')!
+      .get('customer')!
```

Comment:
While we're at it and replacing usages of form getters - we should also make sure we use them where they are relevant.
```suggestion
      this.customer
```